src/main/java/com/endava/internship/mocking/repository/InMemPaymentRepository.java

```diff
@@ -5,12 +5,7 @@
 import static java.util.Optional.ofNullable;
 import static java.util.stream.Collectors.toList;
 
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.NoSuchElementException;
-import java.util.Optional;
-import java.util.UUID;
+import java.util.*;
```

Comment:
please don't use wildcard import

https://blog.marcnuri.com/intellij-idea-how-to-disable-wildcard-imports#:~:text=IntelliJ%20IDEA%20uses%20wildcard%20imports,and%20imports%20the%20full%20package.

Reply:
resolved

Reply:
double check please
![image](https://user-images.githubusercontent.com/80094766/195262803-cac7bf17-cfcd-480b-99b0-4afb0a287c2f.png)